src/APIdata.js

```diff
@@ -0,0 +1,40 @@
+const ApiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+
+const MakeGame = () => {
+  fetch(
```

Comment:
- [ ] From the requirements of this project, we are supposed to send and receive data asynchronously so please let's add `async` and `await` together with the fetch method.

Reply:
fixed

Reply:
resolved